docs/welcome/glossary.md

```diff
@@ -33,6 +37,10 @@ A contributor who has not been active on a project in the last 30 days.
 
 The Contributor Insights feature enables you to categorize, monitor, and analyze different groups of contributors within open source projects. With the Contributor Insights feature, gain granular insights into each contributor's activity and contributions.
 
+## Lottery Factor
+
+The Lottery Factor is a metric that identifies how at risk a project is if a key contributor leaves. It is calculated by the percentage of contributions made by the top contributors. If 50% of the contributions come from two or fewer contributsors, the lottery factor is high. (The name comes from the idea that if a key contributor wins the lottery and leaves, the project is at risk. This is sometimes known as the bus factor.)
```

Comment:
There a few of things here:

- There is a typo
- Not everyone knows the meaning of "bus factor". We might want to to link this to Wikipedia.
Understanding what "bus factor" is would give a background why we came up with the positive term of "lottery factor".
- I think we can make this 2 paragraphs instead of put the origin of the name inside brackets. This worth to have its own place here.

```suggestion
The Lottery Factor is a metric that identifies how at risk a project is if a key contributor leaves. It is calculated by the percentage of contributions made by the top contributors. If 50% of the contributions come from two or fewer contributors, the lottery factor is high. 

The name "Lottery Factor" comes from the idea that if a key contributor wins the lottery and leaves, the project is at risk. This is sometimes known as the [bus factor](https://en.wikipedia.org/wiki/Bus_factor).
```